.github/CONTRIBUTING.md

```diff
@@ -90,6 +90,16 @@ lives in the spot where the go cli tools are expecting to find it.
 
 ## Testing
 
+### During Development: Run Relevant Test(s)
+
+During development, it may be more convenient to check your work-in-progress by running only the tests which you expect to be affected by your changes, as the full test suite can take several minutes to execute. [Go's built-in test tool](https://golang.org/pkg/cmd/go/internal/test/) allows specifying a list of packages to test and the `-run` option to only include test names matching a regular expression.
+
+Examples (run from the repository root):
+- `go test -v ./connect` will run all tests in the connect module (see `./connect` folder)
+- `go test -v ./command/agent -run TestRetryJoin` will run all tests in the agent module (see `./command/agent` folder) with name substring `TestRetryJoin`
```

Comment:
```suggestion
- `go test -v -run TestRetryJoin ./command/agent` will run all tests in the `command/agent` package with name substring `TestRetryJoin`
```

Reply:
I also changed the order of the args. I believe the `-run` arg must come before the package list.

Reply:
Turns out it works either way - I had been running the commands this way while working on my first bugfix. But I can see why that wouldn't be expected (flags don't usually come at the end like that), so I 100% agree with this change.